### ISSUE TITLE
SRB2 Kart Window missing

### ISSUE THREAD

**Issue by TheCodeKirby** [flathub/org.srb2.SRB2Kart#31]
I recently downloaded SRB2 Kart on my Chrombook, using the Linux terminal. Everything went well, I even saw the game in the linux programs folder. I tried running the game from that folder, I started hearing game audio, but no window came up. I looked in the chromebook apps bar, and it didnt show that it was running. I could hear the intro and the title screen music, but couldn't interact with the game. I tried opening it using the terminal, but, to no avail- I got the same results.
Is there something I am doing wrong? I am pretty sure that I installed Flatpak correctly.

**Comment by ghisvail** [comment]
How did you setup Linux on your Chromebook?

**Comment by Lonsfor** [comment]
> I tried running the game from that folder

i dont think you are supposed to do that. try using the application menu (or whatever Chrombook uses to launch apps)

**Comment by TheCodeKirby** [comment]
> > I tried running the game from that folder
> 
> i dont think you are supposed to do that. try using the application menu (or whatever Chromebook uses to launch apps)
When you create a Linux profile on Chromebook, a Linux apps folder appears in the application menu. I ran the game from there, and as previously stated, it didn't work. Thanks for trying to help, though.

**Comment by ghisvail** [comment]
Could you confirm whether it's still happening with the latest release?
